templates/CRM/Case/Form/ActivityTab.tpl

```diff
@@ -31,12 +31,12 @@
         </tr>
         <tr>
           <td class="crm-case-caseview-form-block-activity_date_low">
-            {assign var=activitylow  value=activity_date_low_$caseID}
+            {assign var=activitylow  value="activity_date_low_`$caseID`"}
```

Comment:
I feel like these backticks were a problem last I tried smarty 3 - I'm gonna fish out my code to get it to mostly-load & check

Reply:
Looks like they are not actually required, so we can lose them if you prefer, but they should work and I think it's better to have them because you need them if you have something like `$foo.bar`: https://www.smarty.net/docsv2/en/language.syntax.quotes.tpl

What doesn't work is if you enclose the string in single quotes with a variable - needs to be double quotes if you have a variable.

Reply:
@larssandergreen so it's gonna take me a bit to fish out whatever I did last time that worries me about backticks

Perhaps we could get the other changes merged so they don't go stale while I fish out back tick stuff

Reply:
Oh, I'll deal with merge conflicts if needed, not a big deal.